### ISSUE TITLE
Steam patches and buttons work but no overlay change steam beta

### ISSUE THREAD

**Issue by ripplingsnake** [Maclay74/steam-patch#19]
just a heads up steam0patch still works on steam beta, well the buttons do but the overlay is getting over ridden by steam , hopefully someone can look into the issue and resolve it

**Comment by Maclay74** [comment]
Overlay, you mean, in-game?

**Comment by ripplingsnake** [comment]
Yes steam patches fine for button control but you still have Xbox button bottom left and the tdp slider will stay at standard 15watts , this is only on steam deck beta not the stable version which works fine

**Comment by Maclay74** [comment]
Alright, I'll check it out on weekend, I think.

**Comment by ripplingsnake** [comment]
Perfect steam patch when it works is far less laggy than the handycon implementation that’s why I persevere with it but I can’t code so I can’t help

**Comment by ripplingsnake** [comment]
On nobara we had issues with the patch applying as verification complete is sometimes not the last line of it looks for updates , is there a more robust way of applying the patch ? Regards Marc Gibb On 23 Oct 2023, at 18:55, MIkhail Kozlov ***@***.***> wrote:﻿
Alright, I'll check it out on weekend, I think.

**Comment by ripplingsnake** [comment]
Look like the steamos beta client became stable and has broken steam-patch ui but buttons still work, the ui elements remain steam standard now let’s hope there is a fix
